rtest/package.xml

```diff
@@ -11,12 +11,13 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>ament_cmake_ros</buildtool_depend>
-  <depend>rclcpp</depend>
+  <build_depend>libboost-dev</build_depend>
```

Comment:
Please add `<build_depend>google-mock</build_depend>`
It will not work until the `rosdep` mapping is fixed though.

Reply:
There is also a `libgmock-dev` key. Which worked fine!

Reply:
Oh, cool :)